### PR TITLE
Add strike to lending

### PR DESCRIPTION
**Pull request by tomfutago**

### CODE REVIEW

File: models/_sector/lending/borrow/ethereum/_schema.yml [hunk 1]
```diff
@@ -475,3 +475,46 @@ models:
...
+          seed_file: ref('strike_ethereum_base_borrow_seed')
+          match_columns:
+            - transaction_type
+            - token_address
+            - tx_hash
+            - evt_index
+          check_columns:
+            - block_number
+            - borrower
+            - amount
```
Comment (Hosuke) [inline models/_sector/lending/borrow/ethereum/_schema.yml:505]: Maybe we can make a `check_base_borrow_seed` generic test.

File: seeds/_sector/lending/_schema.yml [hunk 2]
```diff
@@ -923,3 +923,32 @@ seeds:
...
+        version: varchar
+        block_number: bigint
+        block_time: timestamp
+        tx_hash: varbinary
+        evt_index: bigint
+        transaction_type: varchar
+        token_address: varbinary
+        borrower: varbinary
+        amount: double
+  - name: strike_ethereum_base_supply_seed
```
Comment (Hosuke) [inline seeds/_sector/lending/_schema.yml:941]:
```suggestion
        amount: double

  - name: strike_ethereum_base_supply_seed
```

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @tomfutago 

[Sample strike_ethereum_base_borrow]()
[Sample strike_ethereum_base_supply]()

File: models/_sector/lending/borrow/ethereum/_schema.yml (hunk 1 above)
Comment (jeff-dude) [reply models/_sector/lending/borrow/ethereum/_schema.yml:505]: not a requirement, but good suggestion. won't leave that as a blocker. i believe `nft.trades` may be setup similar to here.

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**